### ISSUE TITLE
Misleading "super" tests in generator.declareMethod.{argSuperCall,bodySuperCall}

### ISSUE THREAD

**Issue by anba** [tc39/test262#178]
generator.declareMethod.argSuperCall.js:
> *foo(a = super) {

generator.declareMethod.bodySuperCall.js:
> super;

These lines should use `super()` instead of bare `super` (Arguments list is mandatory after `super`). 

```
js> ({*g() { super; }})
typein:1:15 SyntaxError: invalid token ";"
typein:1:15 ({*g() { super; }})
typein:1:15 ..............^
js> ({*g() { super(); }})
typein:2:15 SyntaxError: 'super()' expression outside of class constructor
typein:2:15 ({*g() { super(); }})
typein:2:15 ..............^
```

**Comment by smikes** [comment]
Thanks.  WIll be fixed by PR #180

**Comment by anba** [comment]
Fixed in 1b04b7db089a8d5f717518d3033207751c744556
